lib/wings/active_fedora_converter.rb

```diff
@@ -100,10 +100,10 @@ class DefaultWork < ActiveFedora::Base
       property :ordered_authors, predicate: ::RDF::Vocab::DC.creator
       property :ordered_nested, predicate: ::RDF::URI("http://example.com/ordered_nested")
       property :nested_resource, predicate: ::RDF::URI("http://example.com/nested_resource"), class_name: "Wings::ActiveFedoraConverter::NestedResource"
+      include ::Hyrax::BasicMetadata
```

Comment:
👍 I found this also in https://github.com/samvera/hyrax/pull/3733/files#diff-01ac1642e5ca1afd32dd5eef2a819d22L103